{
   "assignee" : null,
   "body" : "I generated block 100283 and collected a sub bitcent transaction fee.\nI get an \"Insufficient funds\" error when trying to send my exact balance to another address.\nIt seems that there is either :\n\n* wrong behaviour (should send the amount), or\n\n* wrong error message, (the balance is ok but the client forbids me to send it)\n\nBlock : 000000000003d014210dcffd602715545bce976f0f257d99279c83c2778e7150\n\nHere's the trace\n\n<pre>david@box:~$ bitcoin getinfo\n{\n    \"version\" : 31900,\n    \"balance\" : 50.00638641,\n    \"blocks\" : 100760,\n    \"connections\" : 48,\n    \"proxy\" : \"\",\n    \"generate\" : true,\n    \"genproclimit\" : -1,\n    \"difficulty\" : 14484.16236123,\n    \"hashespersec\" : 9273499,\n    \"testnet\" : false,\n    \"keypoololdest\" : 1291071649,\n    \"paytxfee\" : 0.00000000,\n    \"errors\" : \"\"\n}\ndavid@box:~$ bitcoin sendtoaddress 12DidCRRT9fqmSBReRNN8KjCktYGTVU1Jj 50.00638641\nerror: {\"code\":-4,\"message\":\"Insufficient funds\"}</pre>",
   "closed_at" : "2011-03-22T23:46:46Z",
   "closed_by" : {
      "avatar_url" : "https://avatars.githubusercontent.com/u/345092?v=3",
      "events_url" : "https://api.github.com/users/tcatm/events{/privacy}",
      "followers_url" : "https://api.github.com/users/tcatm/followers",
      "following_url" : "https://api.github.com/users/tcatm/following{/other_user}",
      "gists_url" : "https://api.github.com/users/tcatm/gists{/gist_id}",
      "gravatar_id" : "",
      "html_url" : "https://github.com/tcatm",
      "id" : 345092,
      "login" : "tcatm",
      "organizations_url" : "https://api.github.com/users/tcatm/orgs",
      "received_events_url" : "https://api.github.com/users/tcatm/received_events",
      "repos_url" : "https://api.github.com/users/tcatm/repos",
      "site_admin" : false,
      "starred_url" : "https://api.github.com/users/tcatm/starred{/owner}{/repo}",
      "subscriptions_url" : "https://api.github.com/users/tcatm/subscriptions",
      "type" : "User",
      "url" : "https://api.github.com/users/tcatm"
   },
   "comments" : 2,
   "comments_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/18/comments",
   "created_at" : "2011-01-02T23:40:33Z",
   "events_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/18/events",
   "html_url" : "https://github.com/bitcoin/bitcoin/issues/18",
   "id" : 502368,
   "labels" : [
      {
         "color" : "FBBAAB",
         "name" : "Bug",
         "url" : "https://api.github.com/repos/bitcoin/bitcoin/labels/Bug"
      }
   ],
   "labels_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/18/labels{/name}",
   "locked" : false,
   "milestone" : null,
   "number" : 18,
   "repository_url" : "https://api.github.com/repos/bitcoin/bitcoin",
   "state" : "closed",
   "title" : "Error when trying to send very precise amount",
   "updated_at" : "2014-09-11T05:04:53Z",
   "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/18",
   "user" : {
      "avatar_url" : "https://avatars.githubusercontent.com/u/33329?v=3",
      "events_url" : "https://api.github.com/users/davout/events{/privacy}",
      "followers_url" : "https://api.github.com/users/davout/followers",
      "following_url" : "https://api.github.com/users/davout/following{/other_user}",
      "gists_url" : "https://api.github.com/users/davout/gists{/gist_id}",
      "gravatar_id" : "",
      "html_url" : "https://github.com/davout",
      "id" : 33329,
      "login" : "davout",
      "organizations_url" : "https://api.github.com/users/davout/orgs",
      "received_events_url" : "https://api.github.com/users/davout/received_events",
      "repos_url" : "https://api.github.com/users/davout/repos",
      "site_admin" : false,
      "starred_url" : "https://api.github.com/users/davout/starred{/owner}{/repo}",
      "subscriptions_url" : "https://api.github.com/users/davout/subscriptions",
      "type" : "User",
      "url" : "https://api.github.com/users/davout"
   }
}
